src/platform/backends/lxd/lxd_virtual_machine_factory.cpp

```diff
@@ -111,6 +111,13 @@ void mp::LXDVirtualMachineFactory::hypervisor_health_check()
     }
 }
 
+QString mp::LXDVirtualMachineFactory::get_backend_version_string()
+{
+    auto reply = lxd_request(manager.get(), "GET", base_url);
+
+    return reply["metadata"].toObject()["environment"].toObject()["server_version"].toString();
```

Comment:
This needs to be prefixed with `lxd-`, otherwise:

```
multipass/driver/4.3 # written by Multipass
```

Reply:
Ack, will do.

Reply:
Ok, done.